source/Willow.package/WebComponentInteraction.class/instance/asynchronousCallToAffect..st

```diff
@@ -1,9 +1,10 @@
 accessing - private
 asynchronousCallToAffect: aCanvas
 
-	| asynchronousCall |
-	asynchronousCall := aCanvas jQuery ajax.
-	priorityActions do: [ :action | action value: asynchronousCall value: aCanvas ].
-	self complete: asynchronousCall withResponseOn: aCanvas.
-	self complete: aCanvas withErrorHandlingOn: asynchronousCall.
-	^ asynchronousCall
\ No newline at end of file
+	| serverCall |
+
+	serverCall := aCanvas jQuery serverCall.
```

Comment:
Why not something like `aCanvas jQuery willow serverCall` ? And this encapsulates the setOwner: thing.